packages/victory-canvas/src/point.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { assign } from "lodash";
+import { Helpers, CommonProps } from "victory-core";
+import { useCanvasContext } from "./hooks/use-canvas-context";
+
+const evaluateProps = (props) => {
+  /**
+   * Potential evaluated props are:
+   * `ariaLabel`
+   * `desc`
+   * `id`
+   * `size`
+   * `style`
+   * `symbol`
+   * `tabIndex`
+   */
+  const ariaLabel = Helpers.evaluateProp(props.ariaLabel, props);
+  const desc = Helpers.evaluateProp(props.desc, props);
+  const id = Helpers.evaluateProp(props.id, props);
+  const size = Helpers.evaluateProp(props.size, props);
+  const style = Helpers.evaluateStyle(props.style, props);
+  const symbol = Helpers.evaluateProp(props.symbol, props);
+  const tabIndex = Helpers.evaluateProp(props.tabIndex, props);
+
+  return assign({}, props, {
+    ariaLabel,
+    desc,
+    id,
+    size,
+    style,
+    symbol,
+    tabIndex
+  });
+};
+
+const Point = (initialProps) => {
+  const { canvasRef } = useCanvasContext();
+  const props = evaluateProps(initialProps);
+  const { x, y, style, size } = props;
+
+  const draw = React.useCallback(
+    (ctx) => {
+      ctx.fillStyle = style.fill;
+      ctx.beginPath();
+      ctx.arc(x, y, size, 0, 2 * Math.PI);
```

Comment:
It looks like this is confined to circle points only so far. It would be cool to eventually be ablr to translate from arbitrary svg paths to canvas commands. Basically the reverse of what `d3-path` does: https://github.com/d3/d3-path/blob/main/src/path.js

Reply:
Yes! I have been looking into using something like d3-shape for this. It was pretty cool how we could use the exact same logic for the lines and just set the context to a canvas container instead of SVG, and I'm not sure yet whether there is a way to do this for other shapes.